crates/bevy_render/src/render_graph/nodes/camera_node.rs

```diff
@@ -90,7 +90,7 @@ pub fn camera_node_system(
         render_resource_context.map_buffer(staging_buffer, BufferMapMode::Write);
         staging_buffer
     } else {
-        let size = std::mem::size_of::<[[f32; 4]; 4]>();
+        let size = std::mem::size_of::<[[f32; 4]; 5]>();
```

Comment:
I'd prefer calculating the project and position and then doing `let buffer_size = projection.byte_len() + position.byte_len()` for clarity.

Reply:
The link to the snippet shown is dead for me: do you have a fresh one?

Reply:
Odd, because the snippet is still visible in "Changed Files". But this should work https://github.com/bevyengine/bevy/blob/cea09e982d369172ca905144646d0fccc52421b4/crates/bevy_render/src/render_graph/nodes/camera_node.rs#L90

Reply:
For this too, look at https://github.com/bevyengine/bevy/pull/1554/commits/45d1137b78504054050c3f9bf701751c695e29d9